pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/agent/impl/DockerPipelineFromDockerfileScript.groovy

```diff
@@ -77,7 +77,8 @@ public class DockerPipelineFromDockerfileScript extends AbstractDockerPipelineSc
             try {
                 def hash = Utils.stringToSHA1(script.readFile("${describable.getDockerfilePath()}"))
                 def imgName = "${hash}"
-                script.sh "docker build -t ${imgName} -f \"${describable.getDockerfilePath()}\" \"${describable.getActualDir()}\""
+                def additionalBuildArgs = describable.getAdditionalBuildArgs() ? " ${describable.additionalBuildArgs}" : ""
+                script.sh "docker build -t ${imgName}${additionalBuildArgs} -f \"${describable.getDockerfilePath()}\" \"${describable.getActualDir()}\""
```

Comment:
btw is this really using the same docker executable logic as docker-pipeline-plugin? I thought it used some logic to check for tool installer first, this also does not seem to care about the `withDockerRegistry` stuff.